### ISSUE TITLE
[CodeGenerator] Object reference not set to an instance of an object

### ISSUE THREAD

**Issue by Neikice** [sschmid/Entitas-CSharp#521]
I run "Jenny fix" first. And then I run "Jenny doctor". The doctor show me the error message
"Object reference not set to an instance of an object".
How to fix this??

![snipaste_20171216_162706](https://user-images.githubusercontent.com/783739/34068865-9a19f016-e27e-11e7-938f-5d539bc1bfcb.png)
my.properties
![snipaste_20171216_162815](https://user-images.githubusercontent.com/783739/34068892-495d9b2c-e27f-11e7-9fcf-cf9ea1e25709.png)

**Comment by sschmid** [comment]
Hi,

can you run `jenny doctor -v`
`-v` will result in verbose output and will help identifying the problem

**Comment by Neikice** [comment]
Hi, According to your suggestion, I got the information in the above picture

**Comment by Neikice** [comment]
![snipaste_20171216_223600](https://user-images.githubusercontent.com/783739/34071611-6a7edb20-e2b3-11e7-8eac-7a05968e944c.png)

Hi, According to your suggestion, I got the information in the above picture

**Comment by sschmid** [comment]
Thanks! I'd be very interested if this issue is related to #503. 

tl;dr;
Try removing this line from your -csproj file if it exists
`<TargetFrameworkProfile>Unity Subset v3.5</TargetFrameworkProfile>`

It might reappear, but I'm working on a fix.
I hope this helps already

**Comment by Neikice** [comment]
I have tried this ,yesterday.It is good in my laptop, but not on the desktop.
I'll be waiting for your next  release.

**Comment by Neikice** [comment]
fix in 0.46.3.
